src/Freebox.cpp

```diff
@@ -297,15 +302,14 @@ bool Freebox::StartSession ()
     #define HOST_NAME_MAX 256
   #endif
 #endif
-      char hostname [HOST_NAME_MAX + 1];
```

Comment:
Thanks to `GetHostname`, `HOST_NAME_MAX` can be removed entirely :)

Reply:
Ah yes, them no more need then :+1: becomes removed.

Reply:
Is done.